Formula/cod.rb

```diff
@@ -0,0 +1,18 @@
+class Cod < Formula
+  desc "Completion daemon for bash/fish/zsh"
+  homepage "https://github.com/dim-an/cod"
+  url "https://github.com/dim-an/cod/archive/9096c4bd0b4a34121f35c323e174a139c30aa8f6.tar.gz"
+  sha256 "dfc8de77d33d3a4e4af46eda3b61e4dbbeddf099d8f85bfc0b4c4a2d693b425c"
+  license "Apache-2.0"
+
+  depends_on "go" => :build
+
+  def install
+    system "make", "build"
+    bin.install "cod"
+  end
+
+  test do
+    system bin/"cod", "--help"
```

Comment:
Can you propose a more substantial test than checking the version or calling for help? These tests rarely ever fail even for broken installations.

>We want tests that don’t require any user input and test the basic functionality of the application. For example `foo build-foo input.foo` is a good test and (despite their widespread use) `foo --version` and `foo --help` are bad tests. 

Ref. https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula

Reply:
Good call! Will do.